events/config.go

```diff
@@ -19,6 +21,9 @@ type CloudEventsConfig struct {
 	KafkaVersion          string   `envconfig:"KAFKA_VERSION" required:"true"`
 	KafkaUsername         string   `envconfig:"KAFKA_USERNAME" required:"false"`
 	KafkaPassword         string   `envconfig:"KAFKA_PASSWORD" required:"false"`
+	KafkaDelay            int      `envconfig:"KAFKA_DELAY" default:"0"`
+	CascadeDelays         []int    `envconfig:"KAFKA_CASCADE_DELAYS" default:""`
```

Comment:
I think envocnfig supports parsing `time.Duration` out of the box, so it should be possible to use it directly, instead of `int`. 

Should we provide sensible defaults?